content/browser/xr/service/browser_xr_runtime_impl.cc

```diff
@@ -219,10 +219,6 @@ bool BrowserXRRuntimeImpl::SupportsNonEmulatedHeight() const {
       return false;
     case device::mojom::XRDeviceId::WVR_DEVICE_ID:
       return true;
-#if BUILDFLAG(ENABLE_GVR_SERVICES)
-    case device::mojom::XRDeviceId::GVR_DEVICE_ID:
-      return true;
-#endif  // ENABLE_GVR_SERVICES
```

Comment:
Correct, it seems like Chromium switched to using ARCore and OpenXR completely, all mentions of GVR have been removed.

Reply:
The question is, this is not a wolvic file, why they haven't removed it? Why do we need to do that?

Reply:
You're right, I think it's an artifact of conflict resolution actually. I think I should've removed it when rebasing the original change. I will go back and do that and then rebase the wolvic_update_m123_base branch so this will not be present in this PR.